ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/tree/TomlValue.kt

```diff
@@ -132,10 +133,6 @@ internal constructor(
                             "Control characters (excluding tab) are not permitted" +
                                     " in literal strings."
                         )
-                    '\\' in this ->
```

Comment:
was this intentional?

Reply:
Yep, not sure why it was there to begin with tbh. Literal strings are supposed to interpret "\\" as a normal character rather than an escape, except our non-spec single quote escapes which are handled already

Reply:
Could you please add your example to tests? Looks like I missed that